rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,148 @@
+
+## Author(s)
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+
+## Shepherd
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+
+## Motivation
+
+There is currently no clear and agreed upon definition of the requirements for Data Citation in the DCP.
+Key issues that this RFC seeks to resolve are:
+  - What is the minimum feature set required for a first release (Phase 1)?
+  - What are the discrete set of features that make sense for subsequent releases (e.g. Phase 2, Phase 3, etc.)
+  - In which phase, if any, is support for a formal digital object identifier ([DOI](https://en.wikipedia.org/wiki/Digital_object_identifier)) required?
+
+### User Stories
+
+1. As a data contributor (e.g. researcher with a pipette), it is essential to have a unique way for my project (consisting of primary and secondary analysis data files and associated metadata files) to be identified so that others can properly cite my work when using my data, and that this citation identifier is available in the DCP Data Portal. Note: This will discourage but not prevent others from using my data without attribution.
+2. As a data consumer (e.g. researcher with a keyboard), I want to be able to view and share a unique citation identifier so that a reader of my manuscript can obtain the data needed to reproduce my results. Anyone can use the citation identifier to view and download all the original cited data and metadata files for a project from the DCP.
+3. As a data consumer, I need a simple way to reference a project in the DCP so that I can fulfill the requirements of a Creative Commons attribution license (CC-BY).
+4. As a data contributor or consumer, I need a way to use the citation identifier to access the output produced by the DCP Matrix Service for the data being cited.
+5. As a data contributor or consumer, I want to be able to update my project and provide versioned views of the data being cited.
+
+## Scientific "guardrails" [optional]
+
+*Describe recommended or mandated review from HCA Science governance to ensure that the RFC addresses the needs of the scientific community.*
+
+## Detailed Design
+
+It is proposed to split the initial implementation into three phases:
+
+### Phase 1
+This is designed to satisfy the minimal set of requirements for User Stories #1 through #4 by providing only "per-project" citations.
+The Data Browser project details page will add a "To cite this project please copy this link" item.
+This "stable non-versioned project URL" will link back to the production site project page using the project's UUID (e.g. https://data.humancellatlas.org/explore/projects/cc95ff89-2e68-4a08-a234-480eca21ce79).
+The URL refers to the “live" view of the project and is therefore subject to additions and updates (e.g. corrections) of data and metadata. However, these are expected to be infrequent and should not affect existing primary data.
```

Comment:
I think the statement "should not affect existing primary data" is overstated.

I think updates will rarely affect primary data (apart from adding new primary data to an existing project) but I don't think it will never affect primary data

Reply:
Changed as suggested.